waiter/src/waiter/core.clj

```diff
@@ -1545,6 +1545,7 @@
                                      pr/wrap-too-many-requests
                                      pr/wrap-suspended-service
                                      pr/wrap-response-status-metrics
+                                     pr/wrap-maintenance-mode
```

Comment:
This needs to be before interstitial. We do not want to show an interstitial for a request that will get 503-ed.